### ISSUE TITLE
Podcast in local

### ISSUE THREAD

**Issue by Vctrsnts** [austozi/poddie#2]
Hello.

First of all, thank you for your application.

But I have a question. Could Poddie work with podcast files that are local?

For example, on the other hand, I download a podcast that I like to save, then put it in a directory inside Poddie and it treats it as if it were another podcast and also creates the RSS.

Is this possible? If so, how should it be done? Or would it be very difficult to implement.

Again, thanks for everything

**Comment by austozi** [comment]
@Vctrsnts No, sorry. The only source input for Poddie is the YAML configuration file.

This will take significant effort to implement. I don't have the necessary time or skill to do it. However, there are other solutions that already offer to generate a podcast feed from local files. For example, check out [Audiobookshelf](https://www.audiobookshelf.org).

There may be other options here: https://github.com/awesome-selfhosted/awesome-selfhosted
